app/scripts/views/settings/settings-file-view.js

```diff
@@ -9,7 +9,6 @@ const Links = require('../../const/links');
 const Format = require('../../util/format');
 const Locale = require('../../util/locale');
 const UrlUtil = require('../../util/url-util');
-const kdbxweb = require('kdbxweb');
```

Comment:
Same as above, I guess, it changes almost everything, please open a new PR then.